.codecov.yml

```diff
@@ -34,3 +34,9 @@ ignore:
   - "supporting-docs"
   - "docker"
   - ".github"
+
+flags:
+  unit:
+    paths:
+      - ".*"
```

Comment:
Maybe worth documenting why this is needed?

Reply:
Added into PR description already. It's very common to practice in `codecov` to flag the coverage into unit and integration or other tests. So I didn't found it reasonable to add some comment here.